ecommerce/products/management/commands/create_fixtures.py

```diff
@@ -0,0 +1,85 @@
+import random
+import decimal
+import time
+import datetime
+
+from django.core.management.base import BaseCommand
+from faker import Faker
+
+from products.models import Product, ProductItem
+from products.base import Languages, ProductType, Rarity, Condition
+
+
+fake = Faker()
+
+def word_name():
+    num_words = random.randint(1, 4)
+    return ' '.join(fake.words(num_words)).title()
+
+def random_description():
+    desc = random.randint(50, 100)
+    return fake.paragraph(desc)
```

Comment:
```
def fake_description(min_num_words=50, max_num_words=100):
    ...
```

Reply:
`fake_description()` -> a paragraph between 50 and 100 words
`fake_description(max_num_words=1000)` -> a paragraph between 50 and 1000 words
`fake_description(min_num_words=10)` -> a paragraph between 10 and 100 words
`fake_description(min_num_words=1, max_num_words=1000)` -> a paragraph between 1 and 1000 words

Reply:
Throw an exception if min is not less than max